FinniversKit/Sources/Components/NMPInfobox/NMPInfoboxView.swift

```diff
@@ -0,0 +1,201 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import SwiftUI
+
+public extension NMPInfoboxView {
+    struct ViewModel: Swift.Identifiable {
+        public struct Interaction {
+            public let title: String
+            public let onTapped: () -> Void
+
+            public init(title: String, onTapped: @escaping () -> Void) {
+                self.title = title
+                self.onTapped = onTapped
+            }
+        }
+
+        public let id = UUID()
+        public let informationType: NMPInfoboxView.InformationType
+        public let title: String
+        public let detail: String
+        public let linkInteraction: Interaction?
+        public let primaryButtonInteraction: Interaction?
+        public let secondaryButtonInteraction: Interaction?
+
+        public init(
+            informationType: NMPInfoboxView.InformationType,
+            title: String,
+            detail: String,
+            linkInteraction: Interaction? = nil,
+            primaryButtonInteraction: Interaction? = nil,
+            secondaryButtonInteraction: Interaction? = nil
+        ) {
+            self.informationType = informationType
+            self.title = title
+            self.detail = detail
+            self.linkInteraction = linkInteraction
+            self.primaryButtonInteraction = primaryButtonInteraction
+            self.secondaryButtonInteraction = secondaryButtonInteraction
+        }
+
+        var titleStyle: Label.Style {
+            .bodyStrong
+        }
+
+        var detailStyle: Label.Style {
+            .body
+        }
+    }
+}
+
+#warning("Og så skal der laves en SwiftUI version som rent faktisk bruger den der Infobox ting")
```

Comment:
Indeed! 😄 For these kinds of comments which are about the intent of the code more than the code itself it might be better to use `// TODO:` and let SwiftLint flag it as a warning.

Reply:
Friday afternoon Peter telling Monday morning Peter what to do 😄 

Gone now ✅